src/gui/systray.h

```diff
@@ -79,7 +76,7 @@ public slots:
     Systray();
     bool _isOpen;
     bool _syncIsPaused;
-    QQmlApplicationEngine *_trayEngine;
```

Comment:
This is the wrong fix IMO. QQmlApplicationEngine is a QObject as is Systray. So the systray instance should be passed as parent to the application engine. This is better done in the ctor initialization list instead of in the body BTW.